### DISCUSSION TITLE
Add a NextJS Example to the example folder

### DISCUSSION THREAD

**Discussion by nikolaischunk** [pmndrs/jotai#2023] [Ideas] [answered]
I would love to see a basic example repository with best practises etc in this Repository!

**Comment by dai-shi** [comment, marked as answer]
Does this help?
https://github.com/vercel/next.js/tree/canary/examples/with-jotai

**Comment by nikolaischunk** [comment]
This is for the `pages` directory tho, what about a `app` directory, nextjs 13 implementation?

**Reply by dai-shi** [reply]
I wonder if nextjs repo has something like that already?
